components/02-elements/minicart-trigger/_minicart-trigger.scss

```diff
@@ -1,17 +1,21 @@
+$minicart-trigger-padding           : 10px !default;
+
 $minicart-trigger-background        : $white !default;
 $minicart-trigger-icon-width        : 30px !default;
 
 $minicart-trigger-counter-font-size : $font-size-extra-small !default;
 $minicart-trigger-counter-background: $purple !default;
 $minicart-trigger-counter-color     : $white !default;
 
+$minicart-trigger-breakpoint-large  : $screen-l !default;
```

Comment:
I wouldn't use separate breakpoints in components

Reply:
deleted